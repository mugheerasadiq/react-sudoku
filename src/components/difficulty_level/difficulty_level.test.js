import React from "react";
import renderer from "react-test-renderer";

import DifficultyLevel from "./Difficulty_level.index";

it("All props are passed", () => {
  const tree = renderer
    .create(<DifficultyLevel setLoader={() => {}} setGrid={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
