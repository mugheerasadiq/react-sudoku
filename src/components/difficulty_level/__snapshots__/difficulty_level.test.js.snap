// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All props are passed 1`] = `
<div
  className="difficulty_wrapper"
>
  <h3
    className="ant-typography"
    onClick={null}
    style={
      Object {
        "WebkitLineClamp": undefined,
        "color": "cadetblue",
        "display": "inline",
        "marginRight": "15px",
      }
    }
  >
    Generate
  </h3>
  <button
    className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#93ef00",
        "color": "white",
      }
    }
    type="button"
  >
    <span>
      Easy
    </span>
  </button>
  <button
    className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#e6da00",
        "color": "white",
      }
    }
    type="button"
  >
    <span>
      Medium
    </span>
  </button>
  <button
    className="ant-btn ant-btn-primary ant-btn-lg ant-btn-block ant-btn-dangerous"
    onClick={[Function]}
    type="button"
  >
    <span>
      Hard
    </span>
  </button>
  <button
    className="ant-btn ant-btn-primary ant-btn-lg ant-btn-block"
    onClick={[Function]}
    type="button"
  >
    <span>
      Random
    </span>
  </button>
  <button
    className="ant-btn ant-btn-default ant-btn-lg ant-btn-block ant-btn-dangerous"
    onClick={[Function]}
    style={
      Object {
        "marginLeft": "10px",
      }
    }
    type="button"
  >
    <span>
      Clear
    </span>
  </button>
</div>
`;
